apps/editing-toolkit/editing-toolkit-plugin/site-editor/index.php

```diff
@@ -38,41 +36,18 @@ function initialize_site_editor() {
 function add_site_editor_menu_item() {
 	add_menu_page(
```

Comment:
Can we remove this now and defer the button insertion to core?

Reply:
It seems to still be hidden on dotcom. 😞   I guess its still part of the gutenberg menu that dotcom hides?

Reply:
Oh right, in core it's initialized within Gutenberg admin menu function, and we are suppressing everything from that action in Dotcom.